docs/dev/json.rst

```diff
@@ -143,10 +143,11 @@ A minimal payload must include
 ``artist_name`` and ``track_name`` elements must be simple strings.
 
 The payload will also include the ``listened_at`` element which must be an integer
-representing the Unix time when the track was listened to. The only exception to this
-rule is when the listen is being played right now and has been retrieved from the
-endpoint to get listens being played right now. The ``listened_at`` element will be
-absent for such listens.
+representing the Unix time when the track was listened to. The minimum acceptable
+value for this field is :data:`~webserver.views.api.LISTEN_MINIMUM_TS` The only
+exception to this rule is when the listen is being played right now and has been
+retrieved from the endpoint to get listens being played right now. The
+``listened_at`` element will be absent for such listens.
```

Comment:
Not sure this is important. It seems to say "now playing items won't have this field because they don't have this field".

```suggestion
value for this field is :data:`~webserver.views.api.LISTEN_MINIMUM_TS`.
playing_now requests should not have a ``listened_at`` field
```

(we already talk about playing_now at the beginning of the doc, and say that listened_at isn't required. this just reinforces it)